packages/babel-plugin/src/types.ts

```diff
@@ -107,6 +107,9 @@ export interface PluginOptions {
   /**
    * Adds a defined prefix to the generated classes' hashes.
    * Useful in micro frontend environments to avoid clashing/specificity issues.
+   *
+   * Avoid mixing this with extraction as this may throw an error if combined with extraction
+   * or `extract: true` in Webpack loaders or Parcel tranformers.
```

Comment:
This documentation isn't relevant, I don't think you can mix Babel + extraction elsewhere, but just covering my bases…